src/promptflow/CHANGELOG.md

```diff
@@ -2,9 +2,14 @@
 
 ## 1.4.0 (Upcoming)
 
+### Features Added
+
+- [Executor] Add @trace decorator to make it possible to log traces for functions that are called by tools.
```

Comment:
@zzn2 please help check if this is as expected.

Reply:
Looks good to me.